database/dtb.ksy

```diff
@@ -0,0 +1,128 @@
+meta:
+  id: dtb
+  title: Flattened Devicetree Format
+  file-extension: dtb
+  application:
+    - Linux
+    - Das U-Boot
+  xref:
+    - wikidata: Q16960371
```

Comment:
That's not right - the `meta/xref` must be an object (map), not a list. The compiler should at least warn about this...

```suggestion
  xref:
    wikidata: Q16960371
```

See the [KSY diagram](https://doc.kaitai.io/ksy_diagram.html) for reference.

Reply:
The compiler didn't warn about it.

Reply:
> The compiler didn't warn about it.

I know. When I wrote

> The compiler should at least warn about this...

I meant that it would make sense to implement such a check in the future. We should issue _at least_ a warning (it's not a real obstacle to proper compilation currently, because the compiler does not make any use of `meta/xref` in the generated code anyway) or maybe a fatal error (but that smells like putting artificial obstacles in user's way, which is not great - warning is probably a much better option).